code/modules/religion/honorbound/honorbound_trauma.dm

```diff
@@ -1,19 +1,17 @@
 
 ///Honorbound prevents you from attacking the unready, the just, or the innocent
-/datum/mutation/human/honorbound
-	name = "Honorbound"
-	desc = "Less of a genome and more of a forceful rewrite of genes. Nothing Nanotrasen supplies allows for a genetic restructure like this... \
-	The user feels compelled to follow supposed \"rules of combat\" but in reality they physically are unable to. \
-	Their brain is rewired to excuse any curious inabilities that arise from this odd effect."
-	quality = POSITIVE //so it gets carried over on revives
-	power = /obj/effect/proc_holder/spell/pointed/declare_evil
-	locked = TRUE
-	text_gain_indication = "<span class='notice'>You feel honorbound!</span>"
-	text_lose_indication = "<span class='warning'>You feel unshackled from your code of honor!</span>"
+/datum/brain_trauma/special/honorbound
+	name = "Dogmatic Compulsions"
+	desc = "Patient feels compelled to follow supposed \"rules of combat\"."
+	scan_desc = "damaged frontal lobe"
+	gain_text = "<span class='notice'>You feel honorbound!</span>"
+	lose_text = "<span class='warning'>You feel unshackled from your code of honor!</span>"
+	/// the ability to declare others as evil!
+	var/obj/effect/proc_holder/spell/pointed/declare_evil/declare = /obj/effect/proc_holder/spell/pointed/declare_evil
```

Comment:
Why do you even hold this reference? cause you don't handle the reference in `on_loss` or `destroy`
It doesn't seem necessary at all